### ISSUE TITLE
Method add_uplink for LIG Uplinks Set does not work with Q ports nor integer ports

### ISSUE THREAD

**Issue by fgbulsoni** [HewlettPackard/oneview-sdk-ruby#228]
### Scenario/Intent
When trying to use the lig_uplink_set method to set a Q port or even using the real integer value of the port, the following error is returned:
` Could not evaluate: Port not supported: Q type not found`

### Environment Details
 - **OneView SDK Version:** 4.2.0

### Steps to Reproduce
Run the add_uplink method using a Q or integer port.
i.e.:
```ruby
OneviewSDK::LIGUplinkSet.new(@client, name: 'test').add_uplink(3, 'Q1:1')
```

### Expected Result
The port specified is added to the uplink set

### Actual Result
It returns errors when both an integer or a Q value port is used.

**Comment by fgbulsoni** [comment]
This is related and reported by the issue https://github.com/HewlettPackard/oneview-puppet/issues/148

**Comment by vranystepan** [comment]
Is this related to #216 ? I did not have time to test it yet but it should be possible to get correct _port number_ from /`rest/interconnect-types`. At least for OneView 3.0 ...

**Comment by fgbulsoni** [comment]
@vranystepan Yep, definitely related. I was just looking into using the Q ports and Integer values though, hadn't crossed my mind using different frames.

**Comment by vranystepan** [comment]
@fgbulsoni Multiple frames situation is pretty easy. Will try to figure out. 
Anyway what do you mean with integer values? Do you mean the final `relativeValue` which can be directly placed to the Logical Interconnect Group data?

**Comment by fgbulsoni** [comment]
> Do you mean the final relativeValue

Yep, I've been asked why it would not work if the correct already converted name is used. Made sense I guess. Definitely not a must and something which may even not be used by users, but again, I've been asked about it..

**Comment by vranystepan** [comment]
@fgbulsoni I've just put a few notes in the #216

**Comment by fgbulsoni** [comment]
> @fgbulsoni I've just put a few notes in the #216

@vranystepan Looks like great work there, should get rid of all these issues indeed. Great job! :100:  :octocat:

**Comment by fgbulsoni** [comment]
Closing since this was fixed by #230.
